defaults.json

```diff
@@ -296,13 +619,88 @@
         "Weasel",
         "Weese",
         "Wenda",
+        "Werlag",
         "Wilbert",
-        "Will Humble",
         "Will",
+        "Will Humble",
+        "Willam",
         "William",
+        "Willit",
+        "Willum",
+        "Wolmer",
         "Woth",
+        "Wulfe",
+        "Wylla",
         "Yandry",
         "Yna",
-        "Zei"
-    ]
+        "Yorkel",
+        "Ysilla",
+        "Zei",
+        "Zekko",
+        "Zollo"
+    ],
+
+    "sentimentlist": {
+        "honorable": 2,
+        "nothing": -1,
+        "fck": -4,
+        "fck u": -4,
+        "coolest": 2,
+        "lord": 3,
+        "princess": 3,
+        "boss": 3,
+        "humble": 2,
+        "smug": -2,
+        "feast": 2,
+        "woo hoo": 3,
+        "junk": -2,
+        "independent": 2,
+        "beast": -3,
+        "cutest": 2,
+        "birth": 1,
+        "birthday": 2,
+        "bald": -1,
+        "against": -2,
+        "stand": 2,
+        "gracias": 2,
+        "limit": -2,
+        "tough": -2,
+        "kidnap": -2,
+        "hang": -2,
+        "present": 2,
+        "fairy": 2,
+        "fairy tale": 2,
+        "lengthy": -1,
+        "finer": 2,
+        "fought": -3,
+        "power": 2,
+        "seize": -2,
+        "spoil": -3,
+        "spoiler": -3,
+        "epic": 2,
+        "my lord": 2,
+        "puke": -2,
+        "pukes": -2,
+        "fruitless": -2,
+        "offence": -2,
+        "RIP": -3,
```

Comment:
Sorry, missed this one. I'd agree with Julien, seems more positive to me. It's basically paying your respects :smile: